README.md

```
@@ -0,0 +1,33 @@
+# ✨ TiLoKit – Lightweight Multi-Framework Project Scaffold Generator
+
+[![Go Version](https://img.shields.io/badge/Go-1.24.4-blue)](https://golang.org/dl/)
+[![License](https://img.shields.io/badge/license-MIT-green)](./LICENSE)
+[![Build](https://img.shields.io/badge/build-passing-brightgreen)]()
```

markdownlint-cli2 (0.17.2)

No empty links

(MD042, no-empty-links)